mm2src/mm2_core/src/mm_ctx.rs

```diff
@@ -7,11 +7,12 @@ use common::{executor::{abortable_queue::{AbortableQueue, WeakSpawner},
              expirable_map::ExpirableMap};
 use futures::channel::oneshot;
 use futures::lock::Mutex as AsyncMutex;
-use gstuff::{try_s, Constructible, ERR, ERRL};
+use gstuff::{try_s, ERR, ERRL};
 use lazy_static::lazy_static;
 use libp2p::PeerId;
 use mm2_event_stream::{controller::Controller, Event, EventStreamConfiguration};
 use mm2_metrics::{MetricsArc, MetricsOps};
+use once_cell::sync::OnceCell;
```

Comment:
Why not [OnceLock](https://doc.rust-lang.org/beta/std/sync/struct.OnceLock.html) from the standard library?

Reply:
ah thanks!, I promise, I didn't know this is stable already..also I've had `OnceCell` on my mind for a long time.

Since the API is basically the same, this should one-liner switch. Thanks again!